composer/trainer/checkpoint.py

```diff
@@ -324,22 +324,22 @@ class CheckpointSaver:
     """Manager for saving state to checkpoint files.
 
     Args:
-        checkpoint_folder (str): The path to store checkpoints in.
-        checkpoint_interval (int): The amount of time units to wait between checkpoints.
-        checkpoint_interval_unit (str): The unit (`"ep"` or `"it"`) that
-            `checkpoint_interval` should be measured in.
+        save_folder (str): The path to store checkpoints in.
+        interval (int): The amount of time units to wait between checkpoints.
+        interval_unit (str): The unit (`"ep"` or `"it"`) that
+            `interval` should be measured in.
```

Comment:
Is there a reason this takes an `int interval` and `str interval_unit` instead of a single `Union[str, Time]` à la [`composer.Trainer(max_duration="5ep")`](https://github.com/mosaicml/composer/blob/dev/composer/trainer/trainer.py#L126)? Seems to me like we should try to keep the interface for time specification consistent.

Reply:
Agreed, but will defer this change to later.